dataset_models/sdo/aia.py

```diff
@@ -402,9 +406,9 @@ def _get_prior_x_filename(self, filename, timestep):
         """
         identifier = self._get_prior_timestep_string(filename, timestep)
         if filename[:5] == "noflr":
-            suffix = "_8chnls_1024_0" + str(12 * timestep) + "m.fthr"
+            suffix = "_8chnls_1024_0" + str(12 * timestep) + "h.fthr"
```

Comment:
Should I be worried about these changes? What are the implications? This should really be in a separate pull request. (same below)

Reply:
It is nothing critical, just a naming change that I'm currently using.  Can we leave it?  The download scripts (once they are committed) will be consistent

Reply:
This is a better time to change this than any in the future. I assume it requires re-downloading data? This is worth putting in the summary at the top of the pull request.

Reply:
I will put it then.  This was a change that happened on the last week of the FDL (blush)